examples/react-native/storybook/index.tsx

```diff
@@ -26,9 +26,9 @@ export function setupStorybook(initStorybook: boolean) {
     // add decorators
     addDecorator(withKnobs);
     addDecorator((Story: any) => (
-      <Screen>
+      <DefaultContainer>
```

Comment:
Think this is fine for now but we should probably decouple `DefaultContainer` from our stories in the long term, which most likely means removing this decorator altogether

Reply:
+1 to decouple decorators from our components. If we change  DefaultContainer  it's fine for our authenticator sub-component stories, but perhaps it shouldn't be applied to all primitives stories.